config/dev.yaml

```diff
@@ -12,6 +12,10 @@ common:
     # Type: str
     database_source: ""
 
+    # If set to false, anytime the url output by database_source changes the program will crash
```

Comment:
Could you mention here that `database_source` is run every `sleep_interval` seconds? That wasn't immediately obvious to me. Also might be worth documenting that this option does nothing if `database_source` is not set.

Nit: "any time".

Reply:
i'm kind of dubious having this option is meaningful. using the option means grouper is RO at best and unusable at worse. and it's cognitive load at configuration/setup time that isn't buying much.i vote to not have this flag.

Reply:
if we get rid of this flag we should also get rid of the code that [tries to reconnect to the db in background_processor](https://github.com/dropbox/merou/pull/414/files#diff-9e79234266f5443175806b67277a40cdR153).

Reply:
Is that necessarily true? What if some error other than a URL change occurs and we want to reconnect to the db?

Reply:
> Is that necessarily true?

is that in reference to removing the reconnection code? my suspicion is recovery from sqlachemy `OperationalError` is fraught and we should restart the process anyways to be sure we recovery correctly.

Reply:
Alright I decided to handle ALL exceptions in both threads by crashing (the current behavior was to raise it, which I'm pretty sure kills the background thread while leaving the rest of the process running)